### ISSUE TITLE
chore: `unasync-cli` should be added like a dev dependency

### ISSUE THREAD

**Issue by leynier** [TelePay-cash/telepay-python#11]
To regenerate the synchronous version of code, `unasync-cli` is necessary.

Example: https://github.com/lugodev/qvapay-python/blob/6a530933ca8252fc95ab4cdacec0ff142618bcb4/pyproject.toml#L41

Also, a dev section about synchronous code generation in the Readme file would be good (https://github.com/lugodev/qvapay-python#for-developers)

**Comment by lugodev** [comment]
Hi @leynier, thanks for your collab! What do you mean by "regenerate the synchronous version of code"?

We're doing it manually. Is there a better way to do it?

**Comment by leynier** [comment]
Yeah, only is necessary to implement the async version, the sync version is auto-generate with `unasync-cli`, as is done in `qvavpay-python`.

References:
- https://github.com/lugodev/qvapay-python#for-developers
- https://github.com/leynier/unasync-cli
- https://github.com/python-trio/unasync

**Comment by lugodev** [comment]
Amazing, thanks for the advice. Will do!
